woocommerce/woocommerce-android
# Add new select image button independent from text field

Closes: #12102


### Description

This PR:
- Moves the button for selecting an image outside of text field and below the "Advice box"
- Reduces the size of the textfield and makes it adjust depending on the input length
- Adds back the delay to the animated scrolling when keyboard is out to ensure the textfield is adjusted to the top of the screen. 


### Testing information


1. Trigger product creation with AI flow
2. Select scan image and check the prompt is filled correctly with the scanned text
3. Check the advice box is displayed above the selected image row
4. Verify the UX is usable on smaller screens

### Images/gif


**Small screen experience**

https://github.com/user-attachments/assets/af88fda2-ba62-40d4-96d9-63f820db72c6

https://github.com/user-attachments/assets/3fcafd75-3e9f-4c39-8ed5-0b15f7beeb95
